besu/src/main/resources/org/hyperledger/besu/launcher/launcher.json

```diff
@@ -0,0 +1,102 @@
+[
+    {
+      "prompt-type" :"LIST",
+      "question":"Which ethereum network would you like to use ?",
```

Comment:
```suggestion
      "question":"Which Ethereum network would you like to use ?",
```

Reply:
Maybe we should have an option to list them all?

Reply:
Done